core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -208,6 +212,7 @@ protected void startMasters(boolean isLeader) {
       connectToUFS();
       mRegistry.start(isLeader);
     } catch (IOException e) {
+      LOG.error("Failed to start master.", e);
```

Comment:
does the propagated exception not get logged? This looks like it would result in logging the same thing twice